packages/fxa-auth-server/test/local/routes/auth-schemes/refresh-token.js

```diff
@@ -91,6 +102,9 @@ describe('lib/routes/auth-schemes/refresh-token', () => {
         headers: {
           authorization: 'Bearer Foo',
         },
+        app: {
```

Comment:
nit: Could be more concisely written like `app: { ua }`, or better yet just declare app that beginning of the test.